.github/workflows/main.yml

```diff
@@ -479,5 +483,39 @@ jobs:
         if: env.HAVE_CODESIGN_IDENTITY == 'true'
         uses: actions/upload-artifact@v3
         with:
-          name: 'obs-macos-${{ matrix.arch }}-notarized'
+          name: 'obs-macos-${{ steps.get_version.outputs.VERSION }}-${{ matrix.arch }}'
           path: '${{ github.workspace }}/${{ env.FILE_NAME }}'
+
+  make_release:
+    name: 'Create and upload release'
+    runs-on: [ubuntu-latest]
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+    needs: [macos_release]
+    steps:
+      - name: Get the version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+
+      - name: 'Download macOS x86_64 artifact'
+        uses: actions/download-artifact@v2
+        with:
+          name: obs-macos-${{ steps.get_version.outputs.VERSION }}-x86_64.dmg
+
+      - name: 'Download macOS arm64 artifact'
+        uses: actions/download-artifact@v2
+        with:
+          name: obs-macos-${{ steps.get_version.outputs.VERSION }}-arm64.dmg
+
+      - name: 'Create Release'
+        id: create_release
+        uses: softprops/action-gh-release@1e07f4398721186383de40550babbdf2b84acfc5
+        env:
+          IS_RC: ${{ contains(steps.get_version.outputs.VERSION, 'rc') }}
```

Comment:
Move this check into the "Get the version" step above and rename it to something like "Get Metadata" (see also here: https://github.com/obsproject/obs-plugintemplate/blob/077615678bc8513ed34d9e3fd290abe56822615e/.github/workflows/main.yml#L339). 

Set the RC flag as a step output, so you don't have to use the environment variable workaround.

Reply:
Done